api/CustomEvent.json

```diff
@@ -225,13 +225,13 @@
           "description": "Available in workers",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "43"
```

Comment:
Confirmed by the date of https://bugs.chromium.org/p/chromium/issues/detail?id=370011#c7 + https://www.chromium.org/developers/calendar